e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc.yml

```diff
@@ -46,6 +46,44 @@ spec:
               value: "30100"
             - name: CLIENT_PORT_MAX
               value: "30105"
+            - name: PMM_AGENT_SERVER_ADDRESS
+              value: monitoring-service
+            - name: PMM_AGENT_LISTEN_PORT
+              value: "7777"
+            - name: PMM_AGENT_PORTS_MIN
+              value: "30100"
+            - name: PMM_AGENT_PORTS_MAX
+              value: "30105"
+            - name: PMM_AGENT_SETUP_NODE_ADDRESS
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: status.podIP
+            - name: PMM_AGENT_SETUP_CONTAINER_ID
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
+            - name: PMM_AGENT_SETUP_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
+            - name: PMM_AGENT_SETUP
+              value: "1"
+            - name: PMM_AGENT_SETUP_FORCE
+              value: "1"
+            - name: PMM_AGENT_SERVER_INSECURE_TLS
+              value: "1"
+            - name: PMM_AGENT_CONFIG_FILE
+              value: /usr/local/percona/pmm2/config/pmm-agent.yaml
+            - name: PMM_AGENT_SERVER_USERNAME
+              value: admin
+            - name: PMM_AGENT_SERVER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: pmmserver
+                  name: internal-monitoring
             - name: DB_TYPE
```

Comment:
Please consider adding:
* PMM_AGENT_SETUP_MACHINE_ID;
* PMM_AGENT_SETUP_NODE_MODEL;
* PMM_AGENT_SETUP_NODE_TYPE;
* PMM_AGENT_SETUP_REGION;
* PMM_AGENT_SETUP_AZ.

See https://confluence.percona.com/x/ppGUAg

Reply:
unfortunately we can not get this info

Reply:
Well, you know for sure that PMM_AGENT_SETUP_NODE_TYPE is "container", for example

Reply:
done